test-requirements.txt

```diff
@@ -5,3 +5,4 @@ nose
 nosexcover
 openstack.nose_plugin
 nosehtmloutput
+mock>=1.0
```

Comment:
mock >= 1.1.Y works well on Python 2.6. 

2 month ago, mock 1.1.X was not compatible with 2.6. But they brought back the compatibility. I don't want to find the exact value of X or Y, the window of incompatibility was very short, like 2 weeks, so it's unlikely we see "the" bad version in the wild.
Mock 1.1 never had any issue with Py27 that why no upper bound was set for Py27